website/templates/tagline.html

```diff
@@ -1,3 +1,20 @@
-<!-- your html code here -->
-<!-- Also called a hero image or a jumbotron before bootstrap 5. This might be helpful: https://getbootstrap.com/docs/5.1/examples/jumbotron/ -->
-<p>Hello, I'm the tagline and I was loaded using ajax from templates/tagline.html</p>
\ No newline at end of file
+<div class="pt-5 px-1 text-center bg-image rounded-3" style="
+    background-image: url('https://www.worldatlas.com/r/w1200-q80/upload/dc/2a/1d/shutterstock-557777728.jpg');
+    
+  ">
```

Comment:
Here I would advise a couple of things, either moving the inline style to the `style.css` file or being careful of formatting the line, since something that should occupy 1/2 lines is using 4

```suggestion
<div class="pt-5 px-1 text-center bg-image rounded-3"
  style="background-image: url('https://www.worldatlas.com/r/w1200-q80/upload/dc/2a/1d/shutterstock-557777728.jpg');">
```

Reply:
done, i've moved it to style.css